SECURITY.md

```diff
@@ -0,0 +1,15 @@
+# Security Policy
+
+## Supported Versions
+
+The Drand team will be actively supporting the most recent minor version of drand.
+Upcoming major or minor releases will be announced with a prerelease at least one week in advance, and will be communicated through the leagueofentropy@googlegroups.com email list.
+
+## Reporting a Vulnerability
+
+To report a security issue, please file an issue in this repository, and indicate that it is a vulnerability or security issue.
+If you believe the issue is sensitive or can cause biasability, denial of service, or compromise of the active network,
+please do not directly provide details, but instead provide a method for contacting you off-github, and the drand
```

Comment:
What about inviting them to send an email to drand@protocol.ai?

Reply:
Yes

Reply:
will get a sec mailing list next week